includes/utilities.inc

```diff
@@ -82,4 +62,55 @@ function islandora_memberofsite_fix_relationships($islandora_object, $relationsh
       islandora_memberofsite_add_relationship_if_not_exists($islandora_object, 'isMemberOfSite', $duplicate, $ns);
     }
   }
+
+  $all_sites = islandora_memberofsite_get_sites();
```

Comment:
Let's check here to ensure that we've run any relevant Drupal update hooks (and new configuration of the cmodel value) before performing this cleanup.